ingest-app/src/cmr/ingest/services/granule_bulk_update_service.clj

```diff
@@ -64,14 +109,12 @@
                    json-body
                    instructions)
                   (catch Exception e
-                    (error "validate-and-save-bulk-granule-update caught exception:" e)
-                    (let [message (.getMessage e)
-                          user-facing-message (if (string/includes? message "GBUT_PN_I")
-                                                "Granule bulk update task name needs to be unique within the provider."
-                                                message)]
+                    (error "An exception occurred saving a bulk granule update request:" e)
+                    (let [message (.getMessage e)]
                       (errors/throw-service-errors
-                       :invalid-data
-                       [(str "error creating granule bulk update task: " user-facing-message)]))))]
+                       :internal-error
+                       [(str "There was a problem saving a bulk granule update request."
+                             "Please try again, if the problem persists please contact cmr@nasa.gov.")]))))]
```

Comment:
Since the exception message is likely from the database directly, we may not want to expose it to the user. If we get to this spot we will likely need to figure out what is causing this and have ops or dev support to diagnose and fix the problem. Error is hidden from the user but available to developers and ops in logging.